index.php

```diff
@@ -172,6 +173,12 @@ function send_response($input_raw) {
                 return;
         }
     }
+	if ($chat_id == $user_id) {	
```

Comment:
Please always build your messages using the build_message function. It's there to reduce complexity.